modules/fedimint-lnv2-client/src/lib.rs

```diff
@@ -193,16 +193,16 @@ pub struct PaymentFee {
 }
 
 impl PaymentFee {
-    pub fn one_percent() -> Self {
+    pub fn ten_promille_plus_50_sats() -> Self {
         PaymentFee {
-            base: Amount::ZERO,
+            base: Amount::from_sats(50),
             parts_per_million: 10_000,
         }
     }
 
-    pub fn half_of_one_percent() -> Self {
+    pub fn five_promille_plus_50_sats() -> Self {
         PaymentFee {
-            base: Amount::ZERO,
+            base: Amount::from_sats(50),
```

Comment:
50 sats for a base fee is huge, very often lightning payments are less than this.

Reply:
This is a maximum fee that the client will allow on a payment by default, you can set any value you prefer as a limit though. The gateway right now actually takes this maximum as a fee, mostly for testing that the maximum fee will actually be accepted by the client. In the future we will make this configurable on the gateways side and the client will automatically use the lower base fee set by the gateway. How low the gateway can go here mostly depends on the fee we take per each note input / output as they are driving the transaction size.

Generally all such fees are set very conservatively right now as they have DOS implications but may be lowered at some point.